TransTest.hs

```diff
@@ -18,61 +18,50 @@ testDotGOutput = do
     putStrLn "=== testDotGOutput"
     assertEq threeSignalsConcept expected
   where
-    expected = ".model out\n"          ++
-               ".inputs A\n"           ++
-               ".outputs B\n"          ++
-               ".internal C\n"         ++
-               ".graph\n"              ++
-               "A0 A+\n"               ++
-               "A+ A1\n"               ++
-               "A1 A-\n"               ++
-               "A- A0\n"               ++
-               "B0 B+\n"               ++
-               "B+ B1\n"               ++
-               "B1 B-\n"               ++
-               "B- B0\n"               ++
-               "C0 C+\n"               ++
-               "C+ C1\n"               ++
-               "C1 C-\n"               ++
-               "C- C0\n"               ++
-               ".marking {A0 B1 C0}\n" ++
-               ".end\n"
+    threeSignalsConcept = doTranslate signs (inputs [a] <> outputs [b] <> internals [c] <> initialise0 [a, c] <> initialise1 [b])
+    a = Signal 0
+    b = Signal 1
+    c = Signal 2
+    signs = [a, b, c]
+    expected = unlines ([".model out", ".inputs A", ".outputs B", ".internal C", ".graph"] ++
+                        ["A0 A+", "A+ A1", "A1 A-", "A- A0", "B0 B+", "B+ B1", "B1 B-", "B- B0"] ++
+                        ["C0 C+", "C+ C1", "C1 C-", "C- C0", ".marking {A0 B1 C0}", ".end"])
```

Comment:
Ah, I didn't mean to compress lines together -- sorry should have explained better :) Something like this:
```haskell
    expected = unlines [ ".model out"
                       , ".inputs A"
                       , ".outputs B"
                       ... ]
```